### PR TITLE
Enable disabled vstest and test verb test.

### PR DESCRIPTION
**Pull request by Faizan2304**
Enable disabled vstest and test verb test.

### CODE REVIEW

File: TestAssets/TestProjects/VSTestDotNetCoreProject/VSTestDotNetCoreProject.csproj [hunk 1]
```diff
@@ -19,14 +18,14 @@
...
       <Version>1.0.5-preview</Version>
     </PackageReference>
-	<PackageReference Include="MSTest.TestAdapter">
+    <PackageReference Include="MSTest.TestAdapter">
       <Version>1.1.4-preview</Version>
     </PackageReference>
-	<PackageReference Include="Microsoft.TestPlatform.TestHost">
-      <Version>15.0.0-preview-20161028-03</Version>
+    <PackageReference Include="Microsoft.NET.Test.Sdk">
+      <Version>15.0.0-preview-20161024-02</Version>
```
Comment (livarcocc) [inline TestAssets/TestProjects/VSTestDotNetCoreProject/VSTestDotNetCoreProject.csproj]: You are just rolling back to an older version? How does this contain a fix to the OSX issue? What are we losing between 28-03 and 24-02?

Comment (Faizan2304) [inline TestAssets/TestProjects/VSTestDotNetCoreProject/VSTestDotNetCoreProject.csproj]: Here we are mimicking user scenario. User will not use Microsoft.TestPlatform.TestHost but Microsoft.NET.Test.Sdk package.

Comment (codito) [inline TestAssets/TestProjects/VSTestDotNetCoreProject/VSTestDotNetCoreProject.csproj]: The fix for OSX issue is only in the newer Microsoft.TestPlatform.CLI package that gets pulled in thru redist.json.

**Comment by MattGertz** [comment]
Approved as an adjunct of dotnet/SDK 331.

**Comment by codito** [comment]
@dotnet-bot test Ubuntu16.04 x64 Release Build

**Comment by codito** [comment]
@dotnet-bot test Ubuntu16.04 x64 Debug Build please

**Comment by codito** [comment]
@dotnet-bot test Windows_NT x64 Release Build
@dotnet-bot Windows_NT x86 Debug Build

**Comment by codito** [comment]
@dotnet-bot Windows_NT x86 Debug Build

**Comment by codito** [comment]
@dotnet-bot test Windows_NT x86 Debug Build

**Pull request merged**